local-tests/setup/tinny-environment.ts

```diff
@@ -33,7 +34,10 @@ export class TinnyEnvironment {
     NETWORK: (process.env['NETWORK'] as LIT_TESTNET) || LIT_TESTNET.LOCALCHAIN,
     DEBUG: process.env['DEBUG'] === 'true',
     REQUEST_PER_KILOSECOND:
-      parseInt(process.env['REQUEST_PER_KILOSECOND']) || 200,
+      parseInt(process.env['REQUEST_PER_KILOSECOND']) ||
+      (process.env['NETWORK'] as LIT_TESTNET) === 'datil-dev'
+        ? 1
```

Comment:
1-per-every-10-seconds seems excessively low, no?

Reply:
It doesn't get used anyway for centralised testnet - it's just there to be pretty :))